packages/backend/src/models/SpaceModel.ts

```diff
@@ -491,17 +476,14 @@ export class SpaceModel {
             ])
             .orderBy([
                 {
-                    column: `saved_queries_versions.saved_query_id`, // TODO remove
-                    // column: `saved_queries.saved_query_uuid`, // TODO add
+                    column: `saved_queries.saved_query_uuid`,
```

Comment:
Not sure if this is a good replacement since this is now ordered alphabetically.
I don't think we need this anymore. Ordering by updated_at should be enough 🤔 what do you think ?